.github/workflows/main.yml

```diff
@@ -61,16 +70,40 @@ jobs:
           fi
 
           git clone --branch "$REPO_BRANCH" --depth 1 https://github.com/dlang/dmd.git ../dmd
-      - name: Install prerequisites
+
+      - name: 'Posix: Install prerequisites'
+        if: runner.os != 'Windows'
         run: cd ../dmd && ${{ runner.os == 'macOS' && 'ci/cirrusci.sh' || 'sudo -E ci/cirrusci.sh' }}
-      - name: Install host compiler
+      - name: 'Windows: Set up MSVC environment'
+        if: runner.os == 'Windows'
+        uses: seanmiddleditch/gha-setup-vsdevenv@v4
+        with:
+          arch: ${{ env.MODEL == '64' && 'x64' || 'x86' }}
+
+      - name: 'Posix: Install host compiler'
+        if: runner.os != 'Windows'
         run: cd ../dmd && ci/run.sh install_host_compiler
+      - name: 'Windows: Install host compiler'
+        if: runner.os == 'Windows'
+        uses: dlang-community/setup-dlang@v1.3.0
+        with:
+          compiler: ldc-latest
```

Comment:
I've tried using `dmd-latest` first, but that led to 2 compiler testsuite failures (`runnable/test18141.sh` and a compilable .sh), on Win64 only (Win32 was fine), consistent after a retry attempt.

Reply:
Is this worth putting in bugzilla?

Reply:
Sure would be, but it's pretty vague without analyzing it further ('DMD master built with DMD v2.106.1 crashes for `runnable/test18141.sh` on Win64 only, but works with an LDC host compiler') - which I definitely don't want. ;)